docs/src/howto-guides/015_installation_without_deb_support.md

```diff
@@ -63,10 +63,6 @@ If you have built the binaries from source you should install them on the target
 
 `thin-edge.io` relies on certain system configuration and systemd process management, when installing from deb package all of that is setup automatically but with manual installation a set of steps has to be performed.
 
-On most Linux distribution it should suffice to execute them as `root` to do the setup, but in some cases (eg, your system uses `useradd` instead of `adduser` package) more detailed instructions are documented:
-
-* [tedge](../../../configuration/debian/tedge/postinst)
-* [tedge-agent](../../../configuration/debian/tedge_agent/postinst)
-* [tedge-mapper](../../../configuration/debian/tedge_mapper/postinst)
```

Comment:
Ah its good that you mentioned that.
It wasn't about the content why i removed it, it was because mdbooks didnt want me to link to paths outside mdbook. But i know how to do it now. I can add these lines again.